doc-src/ch-fromhost_desc.rst

```diff
@@ -95,27 +95,38 @@ Additional arguments
 :code:`--cray-mpi` dependencies and quirks
 ==========================================
 
-The implementation of :code:`--cray-mpi` for MPICH is messy, foul smelling,
-and brittle. It replaces or overrides the open source MPICH libraries
-installed in the container. Users should be aware of the following.
+The implementation of :code:`--cray-mpi` is messy, foul smelling, and brittle.
+It replaces or overrides the MPICH or OpenMPI libraries installed in the
+container. Users should be aware of the following.
+
+NOTE: A given item relates to both MPICH and OpenMPI unless otherwise stated.
```

Comment:
I'd remove this and mark everything explicitly.

Reply:
Will do